Changelog-0.16.0.md

```diff
@@ -128,6 +128,7 @@ __Full Changelog__:
  - Enhancement: Add “IMP Quiz Honest Answers” mini mod (#303)
  - Enhancement: Simplify moving to the next sector in tactical view (windowed mode) (#314)
  - Enhancement: Add “Generous Rebels” mini mod (#305)
+ - Maintenance: Explicitly initialize MovementMode (#635)
```

Comment:
This is a vanilla fix and change.

Reply:
Are you sure? I would classify it as: 1/3 vanilla code quality improvements, 2/3 “follow up of bug fix” code quality improvements.